### PR TITLE
CODIP 623 add previous PYTHONPATH

### PR DESCRIPTION
**Pull request by steraite**
fix CODIP US https://jira.cnes.fr/browse/CODIP-623

### CODE REVIEW

File: pesto-cli/pesto/cli/core/docker_builder.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def _env_variables(self):
         return {
             'PESTO_PROFILE': self.build_config.full_version,
             **self.environments,
-            'PYTHONPATH': self._python_path
+            'PYTHONPATH': ":".join(["$PYTHONPATH", self._python_path])
```
Comment (gpo-geo) [inline pesto-cli/pesto/cli/core/docker_builder.py]: Is it still safe when no PYTHONPATH is present in the environment? We will get a PYTHONPATH starting with `:`. Do you see a way to check if the PYTHONPATH is set in the base image?

**Review by gpo-geo — COMMENTED** [review]
Done my review, only one point I would like to check

File: pesto-cli/pesto/cli/core/docker_builder.py (hunk 1 above)
Comment (steraite) [reply pesto-cli/pesto/cli/core/docker_builder.py]: Probably not, I will find an other way to do that.

Comment (steraite) [reply pesto-cli/pesto/cli/core/docker_builder.py]: I change the way to set the PYTHONPATH. Can you check the new version ? I use [parameter expansion](https://wiki.bash-hackers.org/syntax/pe#use_an_alternate_value) to make it work.

Comment (gpo-geo) [reply pesto-cli/pesto/cli/core/docker_builder.py]: I tested your branch, in the container: `PYTHONPATH -> /usr/local/otb/lib/python::/opt/ws-l1c-to-3d `

Comment (gpo-geo) [reply pesto-cli/pesto/cli/core/docker_builder.py]: You could use a one-liner:  `$PYTHONPATH${PYTHONPATH:+:}/opt/service`

Comment (steraite) [reply pesto-cli/pesto/cli/core/docker_builder.py]: changes done here : https://github.com/CS-SI/pesto/pull/1/commits/1fcf15050862c38303c84ee5501cdabdfac71594

**Review by gpo-geo — APPROVED** [review]
ready to merge

**Pull request merged**